doc/api/globals.md

```diff
@@ -80,18 +87,34 @@ added:
 The `AbortSignal` is used to notify observers when the
 `abortController.abort()` method is called.
 
-#### Static method: `AbortSignal.abort()`
+#### Static method: `AbortSignal.abort([reason])`
 
 <!-- YAML
 added:
   - v15.12.0
   - v14.17.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/40807
+    description: Added the new optional reason argument.
 -->
 
+* `reason`: {any}
 * Returns: {AbortSignal}
 
 Returns a new already aborted `AbortSignal`.
 
+#### Static method: `AbortSignal.timeout(delay)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `duration` {number} The numer of milliseconds to wait before triggering
+  the AbortSignal.
```

Comment:
```suggestion
#### Static method: `AbortSignal.timeout(duration)`

<!-- YAML
added: REPLACEME
-->

* `duration` {number} The number of milliseconds to wait before triggering
  the AbortSignal.
```
Or

```suggestion
#### Static method: `AbortSignal.timeout(delay)`

<!-- YAML
added: REPLACEME
-->

* `delay` {number} The number of milliseconds to wait before triggering
  the AbortSignal.
```
Or this to stay consistent with the spec.

```suggestion
#### Static method: `AbortSignal.timeout(milliseconds)`

<!-- YAML
added: REPLACEME
-->

* `milliseconds` {number} The number of milliseconds to wait before triggering
  the AbortSignal.
```

Reply:
I'd rather keep this argument named `delay`